das_api/src/api/api_impl.rs

```diff
@@ -218,6 +224,7 @@ impl ApiContract for DasApi {
     ) -> Result<Vec<Option<Asset>>, DasApiError> {
         let GetAssets { ids, options } = payload;
 
+        let ids = remove_duplicates_ids(ids);
```

Comment:
also dont need to use generic now cause now we filtering only ids, which is Vec<String>